bugzilla/transport.py

```diff
@@ -18,8 +20,6 @@
     from xmlrpclib import Fault, ProtocolError, ServerProxy, Transport
 # pylint: enable=import-error
 
-import requests
```

Comment:
the idea behind the spacing here is to keep python lib imports grouped, and external deps grouped separately. pylint complains about it in most situations but not here, because of the py2 vs py3 bits I think. I don't think moving it adds anything. Plus the moved imports aren't alphabetized within their group

Reply:
This is just a lintfix that complained to me. Normally in Python code you have imports first, then "from"s and then conditional stuff. They also not necessarily need to be alphabetised. I am leaving it for now.